app/css/main.css

```diff
@@ -1,4 +1,14 @@
 /*
+----------------------------------------------------------------------------
+Copyright (c) 2016--, QIIME development team.
+
+Distributed under the terms of the Modified BSD License.
+
+The full license is in the file LICENSE, distributed with this software.
+----------------------------------------------------------------------------
+*/
```

Comment:
Drop the copyright header for this or move the `hljs` stuff to it's own file. (I'd probably do the latter).